index.html

```diff
@@ -578,7 +703,13 @@ <h3 id="placement-of-group-ruby">Placement of group-ruby</h3>
 <h3 id="placement-of-jukugo-ruby">Placement of Jukugo-ruby</h3>
 
 <p>Jukugo-ruby is placed as follows:</p>
-
+<span class="checkme">
+To align following items to the two step processing method described in 
```

Comment:
I think we can add an id to the `li` element of the two-step processing method and link to it directly.

Reply:
to be done as a part of #22